### DISCUSSION TITLE
Is there an easy way to make grid boxes as circles?

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [quarto-dev/quarto-cli#7163] [Q&A] [answered]
### Description

Hi, 

I was wondering if there is any way to turn the boxes from the grid listing type as circles like in the about pages but for the listing items that are in the grid.

Best.

**Comment by mcanouil** [comment, marked as answer]
Depends what you want exactly, turning images as circle is easy.
<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/07a7b296-f9d3-4131-9f4a-d341a423ede7">

```css
.card-img {
  border-radius: 50%;
}
```

Note that for perfect circle you need square images and possibly a bit more CSS.
For that, I suggest you use your favourite browser developper/inspect mode to figure this out.

**Comment by camilogarciabotero** [comment]
That worked pretty well, thanks!
